src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -1623,24 +1620,61 @@ def update_format(study: ParametricStudy) -> ParametricStudy:
             new_study._material_name = materials[0]
             version = 3
 
+            # add p/v column to the dataframe
+            df = ParametricStudy._add_pv_ratio(df)
+
         # Update the dataframe in the new study
         new_study._data_frame = df
         return new_study
 
+    @staticmethod
+    def _add_pv_ratio(df: pd.DataFrame) -> pd.DataFrame:
+        """Add PV Ratio column to the parametric study.
+
+        Parameters
+        ----------
+        study : ParametricStudy
+            Parametric study to update.
+
+        Returns
+        -------
+        ParametricStudy
+            Updated parametric study.
+        """
+        if ColumnNames.PV_RATIO not in df.columns:
+            scan_speed_index = df.columns.get_loc(ColumnNames.SCAN_SPEED)
+            df.insert(scan_speed_index + 1, ColumnNames.PV_RATIO, None)
+
+        df[ColumnNames.PV_RATIO] = df[ColumnNames.LASER_POWER] / df[ColumnNames.SCAN_SPEED]
+        return df
+
     @save_on_return
-    def add_simulations_from_data_frame(self, df: pd.DataFrame) -> list[str]:
+    def add_simulations_from_data_frame(self, file_path: str | os.PathLike) -> list[str]:
```

Comment:
Let's change the name and access level of this function to `_add_simulations_from_csv`. I don't think it's a function we expect end users to call. We want them to use `import_csv_study()`.